src/config.py

```diff
@@ -0,0 +1,26 @@
+"""Application backend configuration."""
+import os
+import connexion
+from flask_sqlalchemy import SQLAlchemy
+from flask_marshmallow import Marshmallow
+
+_POSTGRES_CONFIG = {
+    "HOST": os.environ.get("PG_HOST"),
+    "DATABASE": os.environ.get("PG_DATABASE"),
+    "PASSWORD": os.environ.get("PG_PASSWORD"),
+    "USERNAME": os.environ.get("PG_USERNAME"),
+}
+
+app = connexion.App(__name__, specification_dir="routes/OpenAPI/", server="gevent")
```

Comment:
That's a problem that i am facing with regards to how connexion does routing, I picked this way of initialisation from a tutorial that I read: https://github.com/realpython/materials/blob/master/flask-connexion-rest-part-4/config.py#L23

The issue is if I declare app in the main server file and then try to import it in the schemas and model, but then back-import schemas and models into other route files it gives a circular import error.

I'll park your suggestion for now and look at it once the endpoints are done and refactor because it will take a lot of time to understand the import cycle and not put these objects here.

Reply:
Sure as circular import error means something we are doing wrong, that needs to fix. See if it's taking lot time add TODO in code and take it up later on.